### PR TITLE
Amend PPP

### PR DESCRIPTION
**Pull request by kyhjonathan**

### CODE REVIEW

**Review by NicholasTanYY — APPROVED** [review]
Overall, LGTM! This PPP looks good

File: docs/team/kyhjonathan.md [hunk 1]
```diff
@@ -25,6 +25,22 @@ Given below are my contributions to the project.
...
+### Contributions to the Developer Guide:
+* Added implementation details for the `Storage` class.
+* Add UML Class and Sequence diagrams for the `Storage` class and addManager.
+* Add in Table of Contents.
+* Ensured headings and font sizes were correct.
+
+### Contributions to team-based tasks:
+* Set up team organisation and repository and ensured that all team members had access to the repository.
+* Incorporated the class for ics development.
+* Maintaining of issue tracking and ensuring that all issues were resolved in a timely manner.
```
Comment (clarencepohh) [inline docs/team/kyhjonathan.md:42]: Your contributions are greatly valued in our team! Great working with you!

File: docs/team/kyhjonathan.md [hunk 2]
```diff
@@ -25,6 +25,22 @@ Given below are my contributions to the project.
...
+
 ### Reflections:
 I have learnt much about the practices of Software Engineering that is undoubtedly paramount in the future.
 Extremely educational and important module that has taught me the importance of proper documentation and testing.
```
Comment (clarencepohh) [inline docs/team/kyhjonathan.md:46]: 👍 Very insightful!

**Review by clarencepohh — APPROVED** [review]
Great contributions!

**Review by Aak242 — COMMENTED** [review]
Overall, this is quite a detailed and well documented PPP! LGTM!

**Pull request merged**